hazelcast-client-new/src/main/java/com/hazelcast/client/spi/impl/ClientNonSmartInvocationServiceImpl.java

```diff
@@ -29,6 +29,7 @@ public void invokeOnConnection(ClientInvocation invocation, ClientConnection con
 
     @Override
     public void invokeOnPartitionOwner(ClientInvocation invocation, int partitionId) throws IOException {
+        invocation.getClientMessage().partitionId(partitionId);
```

Comment:
We generally use `set*` prefix for setter methods in Hazelcast. I think choosing another naming convention is confusing.